src/main/java/org/cactoos/scalar/AvgOf.java

```diff
@@ -83,16 +83,13 @@ public AvgOf(final Integer... src) {
             }
             return sum / total;
         }, () -> {
-            int sum = 0;
```

Comment:
@allanws I don't know if is a good idea change `int`, `long` and `float` calculations for `double`. Is not possible use `int`, `long` and `float`? If not, why not?